app/src/main/java/com/teamzzong/hacker/presentation/battle/BindingConversions.kt

```diff
@@ -0,0 +1,13 @@
+package com.teamzzong.hacker.presentation.battle
+
+import android.widget.ImageView
+import androidx.databinding.BindingAdapter
+import coil.load
+
+object BindingConversions {
+    @BindingAdapter("imageUrl")
+    @JvmStatic
+    fun loadImage(imageView: ImageView, url: String) {
+        imageView.load(url)
+    }
+}
```

Comment:
```suggestion
    @BindingAdapter("imageUrl")
    fun ImageView.loadImage(url: String) {
        imageView.load(url)
    }
```

이정도만 해도 충분할 듯, 그리고 BindingAdapter 자체가 메인스레드에서 비동기로 처리하는 방식이라 조금 느릴 수도 있어. 그래서 이미지 받아오는건 그냥 데이터바인딩 안 사용하는게 더 좋다고 생각해.

Reply:
확인!